WebApp/src/components/RegisterPage/RegisterPage.js

```diff
@@ -102,7 +102,7 @@ class RegisterPage extends Component {
 
 
   handleChange = event => {
-    event.preventDefault();
+    //event.preventDefault();
```

Comment:
why commented

Reply:
It prevented chceckbox to be checked or unchecked on first try.